mmseg/models/losses/cross_entropy_loss.py

```diff
@@ -172,6 +172,28 @@ def __init__(self,
         else:
             self.cls_criterion = cross_entropy
 
+    def _get_class_weight(self, class_weight):
+        """Get class weight for loss function.
+
+        Args:
+            class_weight (list[float] | str | None): If class_weight is a str,
+                take it as a file name and read from it.
+        """
+        import numpy as np
+        import mmcv
+
+        if isinstance(class_weight, str):
+            # take it as a file path
+            if class_weight.endswith('.npy'):
+                class_weight = np.load(class_weight)
+            else:
+                file_format = class_weight.split('.')[-1]
+                assert file_format in ['json', 'yaml', 'pkl'], \
+                    f'unsupported class_weight file type {file_format}'
```

Comment:
This part is already handled by MMCV [here](https://github.com/open-mmlab/mmcv/blob/master/mmcv/fileio/io.py#L37)

Reply:
Great to hear.